src/dipdup/config/tezos_tzkt_operations.py

```diff
@@ -126,6 +126,13 @@ def format_untyped_operation_argument(
             return arg_name, 'TzktOperationData | None'
         return arg_name, 'TzktOperationData'
 
+    @classmethod
+    def format_sr_execute_argument(
+        cls,
+    ) -> tuple[str, str]:
+        arg_name = 'execute'
```

Comment:
Can one handler contain multiple `sr_execute` operations? I'd go with `pascal_to_snake(alias or f'sr_execute_{subgroup_index}')`.

Reply:
It's almost improbable that 2 separate execute_outbox_messages could ever end up in the same block at all.